arch/otbn/machine.s.dfy

```diff
@@ -699,6 +700,7 @@ module ot_machine {
                 case Ins256(ins) => eval_ins256(ins)
                 case Block(block) => eval_block(block)
                 case While(cond, body) => eval_while(body, eval_cond(cond))
+                case Function(name, body) => eval_block(body)
```

Comment:
We need to incorporate the machine-based call stack, right?

Reply:
You are right, I have not done that yet. I have taken the lazy approach of augmenting the `va_lemma_Block`, rather than having `vale` emitting the a separate `va_lemma_Function`, just so that we have something working first.

Reply:
Okay.  Adding a comment with a big TODO would be helpful, since we don't want to lose track of that important soundness check.